modules/navier_stokes/doc/content/source/hdgkernels/NavierStokesHDGKernel.md

```diff
@@ -0,0 +1,21 @@
+# NavierStokesHDGKernel
+
+This class implements the steady incompressible Navier-Stokes equations. These include the conservation of mass:
+
+!equation id=mass-eq
+\nabla \cdot \vec{u} = 0
+
+where $\vec{u}$ is the velocity and conservation of momentum:
+
+!equation id=momentum-eq
+\nabla \cdot \left(\vec{u} \otimes \vec{u}\right) - \nabla \cdot \left(\nu
+\nabla\vec{u}\right) + \nabla p = 0
+
+where $\nu$ is the kinematic viscosity and $p$ is the pressure. This class uses
```

Comment:
```suggestion
where $\nu$ is the kinematic viscosity and $p$ is the density-corrected pressure. This class uses
```

Reply:
Assuming that you want to keep this formulation.

Reply:
changed to use rho and mu